src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -73,17 +76,46 @@ public void mecanumDrive(double joystickX, double joystickY, double rotation) {
 		if (Math.abs(backRightPower) > ControlConstants.MAX_ROBOT_SPEED)
 			backRightPower *= ControlConstants.MAX_ROBOT_SPEED / Math.abs(backRightPower);
 
+
+		//strafing finetuning
+
+		if (Math.abs(joystickX) <= 0.3)
+			joystickX = Math.tan(joystickY);
+		if (Math.abs(joystickX) >= -0.3)
+			joystickX = Math.tan(joystickY);
```

Comment:
After making your changes, test (with the robot) that snap-to-grid works and request a merge review.

Reply:
This makes it impossible to go forward - if the joystickX is 0, it will be reassigned to tan of joystickY - so full power forward only will HAVE strafe instead of eliminating it